buildSrc/src/main/kotlin/com.hedera.hashgraph.aggregate-reports.gradle.kts

```diff
@@ -52,7 +51,7 @@ sonarqube {
     }
 }
 
-tasks.create("githubVersionSummary") {
+tasks.register("githubVersionSummary") {
```

Comment:
For this and other changes. Prefer the following to stick to task configuration avoidance (https://docs.gradle.org/current/userguide/task_configuration_avoidance.html):

- `tasks.register(...)` - for a new task
- `tasks.named(...)` - to configure an existing task
- `tasks.names.contains(...)` - to check if a task with a certain name was registered (if needed)
- `tasks.withType<TaskType>().configureEach { ... }` - configure all tasks of a certain type, f they are part of the current build)

Reply:
I have struggled with when to use register vs create as you can see from the gradle files there is a mix of both. 

When is the `tasks.create()` call actually required?